KlockanAPI/src/KlockanAPI.Application/DTOs/Classroom/CreateClassroomDTO.cs

```diff
@@ -0,0 +1,5 @@
+﻿namespace KlockanAPI.Application.DTOs.Classroom;
```

Comment:
Why empty files?

Reply:
Actually, the CreateClassroomDTO class is on that file, right now is just a placeholder.